### PR TITLE
Remove CentOS Stream 8 configuration leftovers

### PR DESCRIPTION
**Pull request by elfosardo**
CentOS Stream 8 is EoL and we don't really support it. Removing what's left from the configuration.

### CODE REVIEW

File: vm-setup/roles/packages_installation/tasks/main.yml [hunk 1]
```diff
@@ -46,24 +46,17 @@
...
-- name: Install packages on CentOS/RHEL8
+- name: Install packages on CentOS/RHEL
   block:
     - name: Install packages on CentOS/RHEL
       package:
           name: "{{ packages.centos.common.packages }}"
           state: present
-    - name: Install packages on CentOS8/RHEL8
-      package:
-          name: "{{ packages.centos.el8.packages }}"
```
Comment (dtantsur) [inline vm-setup/roles/packages_installation/tasks/main.yml:57]: I suspect we can now consolidate the el8 and el9 lists (at least until el10 is a thing, which is relatively soon lol)

Comment (elfosardo) [reply vm-setup/roles/packages_installation/tasks/main.yml:57]: that's what blocked me from doing it

Comment (dtantsur) [reply vm-setup/roles/packages_installation/tasks/main.yml:57]: yeah, but we still don't need el8 packages any more

File: vm-setup/roles/packages_installation/tasks/main.yml [hunk 2]
```diff
@@ -46,24 +46,17 @@
...
-          state: present
-      when: ansible_distribution_version == "8"
     - name: Install packages on CentOS9/RHEL9
       package:
           name: "{{ packages.centos.el9.packages }}"
           state: present
       when: ansible_distribution_version == "9"
-    - name: Perform CentOS/RHEL8 required configurations
-      include_tasks: centos_required_packages.yml
```
Comment (dtantsur) [inline vm-setup/roles/packages_installation/tasks/main.yml:66]: Can we remove this file now?

Actually, this step is not conditional on version 8, maybe it's still needed?

File: vm-setup/roles/packages_installation/tasks/main.yml (hunk 1 above)
Comment (elfosardo) [reply vm-setup/roles/packages_installation/tasks/main.yml:57]: yep, that's what I forgot, thanks

File: vm-setup/roles/packages_installation/tasks/main.yml (hunk 2 above)
Comment (elfosardo) [reply vm-setup/roles/packages_installation/tasks/main.yml:66]: I thought also it was needed by other centos versions but it looks like only CS8 uses it
I'll remove it

**Review by dtantsur — APPROVED** [review]

**Comment by tuminoid** [comment]
/override metal3-centos-e2e-integration-test-release-1-7

This is not relevant really, skip.

**Pull request merged**